public/sw.js

```diff
@@ -342,38 +451,46 @@ async function cacheDynamicFiles(token, dynamicLinks, mode = 'cors') {
 
   // The promise to return with a boolean value determining if all links cached successfully
   operationSuccess = await new Promise((resolve, reject) => {
-    isSuccessful === true ? resolve(true) : reject(false);
+    isSuccessful === true ? resolve(true) : resolve(false);
```

Comment:
The comments I have there doesn't help me understand on why I did that but if I had to make a guess, I believe that's there because I wanted the Promise to pass no matter what. Looking at the code surrounding it, I believe I created that Promise to force that whole function to wait until the variable "successful" had a value in it instead of skipping it and causing isSuccessful to be returned before the fetches are completed.